planning/behavior_velocity_planner/autoware_behavior_velocity_intersection_module/src/scene_intersection.cpp

```diff
@@ -392,9 +392,10 @@ DecisionResult IntersectionModule::modifyPathVelocityDetail(
     if (!has_traffic_light_) {
       if (fromEgoDist(occlusion_wo_tl_pass_judge_line_idx) < 0) {
         if (has_collision) {
-          const auto closest_idx = intersection_stoplines.closest_idx;
+          const auto closest_stopline_idx = intersection_stoplines.closest_idx;
```

Comment:
closest_idx is already defined, so you can just delete L395. BTW, does cppcheck also report L272 ? That line has same mistakes.

Reply:
Thanks for the advice, I have deleted L395.

No shadowVariable warning is generated for L272 because it is defined there for the first time.
A shadowVariable occurs when there is a variable of the same name in a scope outside the scope of that variable.